cmd/hammy/main.go

```diff
@@ -10,15 +12,36 @@ import (
 	"github.com/bwmarrin/discordgo"
 )
 
-const botTokenEnv = "DISCORD_BOT_TOKEN"
+type Config struct {
+	LlmUrl          string `mapstructure:"LLM_URL"`
+	ResponseEmoji   string `mapstructure:"RESPONSE_EMOJI"`
+	LogLevel        int    `mapstructure:"LOG_LEVEL"`
+	DiscordLogLevel int    `mapstructure:"DISCORD_LOG_LEVEL"`
+	BotToken        string `mapstructure:"DISCORD_BOT_TOKEN"`
+}
+
+func init() {
+	viper.SetDefault("ResponseEmoji", "\U0001F914")
```

Comment:
I just wanted a way to default values that we may want to override later, probably overkill for a good bit of this but thought I was going to reference a few more often. This will work without a formal config file if we just want to keep it in github secrets or however we set it up.

Reply:
Since most of this isn't a secret though maybe a config file is a good way to go for the non sensitive bits